ansibox

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+export ANSIBLE_CONFIG=.ansible.cfg
+":" //#explained here sambal.org/?p=1014; exec ansible-playbook -i hosts .playbook.yaml "$@"
```

Comment:
Its an odd hack... to be honest, I'm not sure its even needed to be a hack. I blame that on chasing a rabbit hole.  I'll try it a simplier way.


That being said, yes, you can supposedly pass additional playbooks, and even add -e variables setups.